### PR TITLE
HeisenCat

### PR DESCRIPTION
**Pull request by FShenoda**
Update image to Heisencat

Fixes: #13

### CODE REVIEW

File: _posts/0000-01-03-FShenoda.md [hunk 1]
```diff
@@ -5,5 +5,5 @@ title: "Welcome to our slide deck!"
 
 Caption here
```
Comment (Gymky) [inline _posts/0000-01-03-FShenoda.md]:
```suggestion
Caption here
```

File: _posts/0000-01-03-FShenoda.md [hunk 2]
```diff
@@ -5,5 +5,5 @@ title: "Welcome to our slide deck!"
```
Comment (Gymky) [inline _posts/0000-01-03-FShenoda.md]: Change title to reflect page information

File: _posts/0000-01-03-FShenoda.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Welcome to HeisenCat's page
```
Comment (FShenoda) [inline _posts/0000-01-03-FShenoda.md]:
```suggestion
Welcome to HeisenCat's page !!!!
```

Comment (FShenoda) [reply _posts/0000-01-03-FShenoda.md]: works

**Pull request merged**